src/sentry/replays/usecases/query/configs/materialized_view.py

```diff
@@ -54,7 +54,7 @@ def activity_expression(count_errors, count_urls):
     "device_name": Function("anyIfMerge", parameters=[Column("device_name")]),
     "dist": Function("anyIfMerge", parameters=[Column("dist")]),
     "duration": Function("anyIfMerge", parameters=[Column("duration")]),
-    "finished_at": Function("maxMerge", parameters=[Column("finished_at")]),
+    "finished_at": Function("maxIfMerge", parameters=[Column("finished_at")]),
```

Comment:
Why do some fields use -IfMerge but others use -Merge? Don't we need to pass a condition as a parameter for -Ifs?

Reply:
You only need the -If suffix if your aggregation function requires it.  Some fields do not have a conditional operator when grouping.  You can use the command `SHOW CREATE TABLE replays_aggregated_local` to see which fields require the -IfMerge suffix.